tools/authors/manage.php

```diff
@@ -132,8 +132,8 @@
     $count = count($enable_author_values);
     for ($i = 0; $i < $count; $i++) {
         $sql = sprintf(
-            "UPDATE authors SET enabled = '%s' WHERE author_id = %d",
-            DPDatabase::escape($enable_author_values[$i]),
+            "UPDATE authors SET enabled = '%d' WHERE author_id = %d",
```

Comment:
We shouldn't surround numbers with quotes, even though MySQL will accept it.
```suggestion
            "UPDATE authors SET enabled = %d WHERE author_id = %d",
```